File: Java/UberBlack.java
import java.util.ArrayList;
import java.utils.Map;

class UberBlack extends Car {
    // Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;


    public UberBlack(String license, Account driver/*, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial */){
        super(license, driver);
        // this.typeCarAccepted = typeCarAccepted;
        // this.seatsMaterial= seatsMaterial;
    }
    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passenger = passenger;
        }else {
            System.out.println("You must assign 4 passengers");
        }
    }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, setPassengers, typeCarAccepted, seatsMaterial) -> None:
        # super().__init__(license, driver)
        super().printCarData(license, driver, setPassengers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
        if setPassengers == 4:
            print('Type Car: ' + str(typeCarAccepted) + '\nSeats: ' + str(seatsMaterial))
        else:
            print('You must assign 4 passengers')